code/core_allvars.c

```diff
@@ -1,38 +1,52 @@
 #include "core_allvars.h"
 
 
-// galaxy data 
+/*  galaxy data  */
 struct GALAXY			
   *Gal, *HaloGal;
 
 struct halo_data *Halo;
 
-// auxiliary halo data 
+/*  auxiliary halo data  */
 struct halo_aux_data		
   *HaloAux;
 
 
-// misc 
+/*  misc  */
+
+int HDF5Output;
+#ifdef HDF5
+char          *core_output_file;
+size_t         HDF5_dst_size;
+size_t        *HDF5_dst_offsets;
+size_t        *HDF5_dst_sizes;
+const char   **HDF5_field_names;
+hid_t         *HDF5_field_types;
+int            HDF5_n_props;
+#endif
+
 int FirstFile;
 int LastFile;
 int MaxGals;
 int FoF_MaxGals;
-int Ntrees;			   // number of trees in current file 
-int NumGals;			 // Total number of galaxies stored for current tree 
+int Ntrees;			   /*  number of trees in current file  */
```

Comment:
All of these were my bad. I am in the process of replacing all `//` style comments to `/* */` style comments.